ColorCode_Main.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+#include <assert.h>
+#include "ColorCode_Main.h"
+

```

Comment:
Good separation of test code and production code. This file has only test code. Think of a more appropriate filename